x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/alerts.ts

```diff
@@ -603,6 +603,140 @@ export default function alertTests({ getService }: FtrProviderContext) {
               throw new Error(`Scenario untested: ${JSON.stringify(scenario)}`);
           }
         });
+
+        it(`shouldn't schedule actions when alert is muted`, async () => {
+          const reference = `create-test-8:${user.username}`;
+          const createdAction = await createIndexRecordAction(space.id);
+          const response = await supertestWithoutAuth
+            .post(`${getUrlPrefix(space.id)}/api/alert`)
+            .set('kbn-xsrf', 'foo')
+            .auth(user.username, user.password)
+            .send(
+              getTestAlertData({
+                enabled: false,
+                interval: '1s',
+                alertTypeId: 'test.always-firing',
+                alertTypeParams: {
+                  index: ES_TEST_INDEX_NAME,
+                  reference,
+                },
+                actions: [
+                  {
+                    group: 'default',
+                    id: createdAction.id,
+                    params: {
+                      index: ES_TEST_INDEX_NAME,
+                      reference,
+                      message: 'from:default',
+                    },
+                  },
+                ],
+              })
+            );
+
+          switch (scenario.id) {
+            case 'no_kibana_privileges at space1':
+            case 'space_1_all at space2':
+            case 'global_read at space1':
+              expect(response.statusCode).to.eql(404);
+              expect(response.body).to.eql({
+                statusCode: 404,
+                error: 'Not Found',
+                message: 'Not Found',
+              });
+              break;
+            case 'space_1_all at space1':
+            case 'superuser at space1':
+              await supertestWithoutAuth
+                .post(`${getUrlPrefix(space.id)}/api/alert/${response.body.id}/_mute`)
+                .set('kbn-xsrf', 'foo')
+                .auth(user.username, user.password)
+                .expect(204, '');
+              await supertestWithoutAuth
+                .post(`${getUrlPrefix(space.id)}/api/alert/${response.body.id}/_enable`)
+                .set('kbn-xsrf', 'foo')
+                .auth(user.username, user.password)
+                .expect(204, '');
+              // Wait until alerts scheduled actions twice to ensure actions had a chance to execute once
```

Comment:
This comment doesn't match the code.

Reply:
Fixed in https://github.com/elastic/kibana/pull/43712/commits/5c147c1a74f36963170504579d67cded32b5fab1.